modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaJAXRSSpecServerCodegen.java

```diff
@@ -127,7 +127,8 @@ public void processOpts() {
             if (!supportAsync) {
                 additionalProperties.remove(SUPPORT_ASYNC);
             } else {
-                setJava8ModeAndAdditionalProperties(true);
+                // java8 tag has been deprecated
```

Comment:
What is the intent of keeping these comments? To remove them after showing a JDK8 upgrade is stable?

Reply:
just leave a mark there what the `else` clause is for.